package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -57,7 +61,10 @@ def load_data(
         from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
 
         with KedroSession.create(
-            project_path=project_path, env=env, save_on_close=False
+            project_path=project_path,
+            env=env,
+            save_on_close=False,
+            extra_params=extra_params,
```

Comment:
We should also make the same change in the two other occurrences of `KedroSession.create` further down this file.